internal/service/organization/data_source_organizations.go

```diff
@@ -22,8 +24,9 @@ func PluralDataSource() *schema.Resource {
 				Optional: true,
 			},
 			"include_deleted_orgs": {
-				Type:     schema.TypeBool,
-				Optional: true,
+				Type:       schema.TypeBool,
+				Optional:   true,
+				Deprecated: fmt.Sprintf(constant.DeprecationParamByDate, "January 2025"),
```

Comment:
@Zuhairahmed could you please confirm the removal date for this as per comment here https://jira.mongodb.org/browse/CLOUDP-219771?focusedCommentId=6007024&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6007024

Reply:
q: if the attribute is no longer usable, would it be better to remove directly? I see the value in deprecating if behaviour is maintained but I believe this is not the case.

Reply:
removing directly would be a breaking change if users are using this in their configs

Reply:
correct, but if they are using `include_deleted_orgs` this functionality would not work anymore because the attribute is not being used when calling the API. Both options would be a breaking change in some form unless we preserve using the old sdk for a deprecation period (but I understand this would not allow to support new attributes in the plural data source).

Reply:
Sorry had missed the part that it was a no-op for atlas, sound goods thanks for clarifying